src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
 // write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+const newLeft = Math.round(wall.offsetWidth - spider.clientWidth) / 2;
```

Comment:
The calculation for newLeft should ensure that the division by 2 is applied to the entire expression. Currently, it divides only the spider's clientWidth by 2. Consider using parentheses to ensure the correct order of operations.